Sources/Clappr_tvOS/Classes/Base/Player.swift

```diff
@@ -29,6 +32,30 @@ open class Player: UIViewController {
 
         core?.render()
     }
+    
+    func addTvRemoveGestureIfNeeded(_ viewController: AVPlayerViewController) {
+        if let tvRemoteGesture = tvRemoteGesture,
+            let viewGestures = viewController.view.gestureRecognizers,
+            viewGestures.contains(tvRemoteGesture) {
+            return
+        }
+        
+        let gestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(handleTvRemoteGesture))
+        gestureRecognizer.allowedPressTypes = [NSNumber(value: UIPress.PressType.playPause.rawValue),
+                                               NSNumber(value: UIPress.PressType.select.rawValue)]
+        
+        viewController.view.addGestureRecognizer(gestureRecognizer)
+        self.tvRemoteGesture = gestureRecognizer
+    }
+
+    @objc func handleTvRemoteGesture() {
+        guard let playback = activePlayback as? AVFoundationPlayback else { return }
+        if playback.isPaused {
```

Comment:
I think that we have to respect the user action, not infer based on `playback` state. I mean, if we're trying to make a pause we should call a pause. 
Make sense?

Reply:
The remote controller has just one button to play/pause and a select button that acts the same. This will respect the user action.